core/cmd/bridge_commands_test.go

```diff
@@ -66,15 +68,15 @@ func TestClient_IndexBridges(t *testing.T) {
 	client, r := app.NewClientAndRenderer()
 
 	bt1 := &bridges.BridgeType{
-		Name:          bridges.MustParseBridgeName("testingbridges1"),
+		Name:          bridges.MustParseBridgeName("cliindexbridges1"),
 		URL:           cltest.WebURL(t, "https://testing.com/bridges"),
 		Confirmations: 0,
 	}
 	err := app.BridgeORM().CreateBridgeType(bt1)
 	require.NoError(t, err)
 
 	bt2 := &bridges.BridgeType{
-		Name:          bridges.MustParseBridgeName("testingbridges2"),
+		Name:          bridges.MustParseBridgeName("cliindexbridges2"),
```

Comment:
cannot use randoms here since the names are returned ordered by names, and using randoms would randomize the order for testing assertions.